eng/config/PublishData.json

```diff
@@ -6,19 +6,19 @@
   "packages": {
     "default": {
       "Microsoft.CodeAnalysis": "arcade",
-      "Microsoft.CodeAnalysis.Common": "arcade",
+      "Microsoft.CodeAnalysis.Common": "vssdk",
       "Microsoft.CodeAnalysis.Compilers": "arcade",
       "Microsoft.CodeAnalysis.Scripting": "arcade",
       "Microsoft.CodeAnalysis.Scripting.Common": "arcade",
       "Microsoft.CodeAnalysis.Workspaces.Common": "arcade",
```

Comment:
So if you're here, we're going to need a few more to completely move the VS build forward:

- Microsoft.CodeAnalysis.Workspaces.Common
- Microsoft.CodeAnalysis.CSharp.Workspaces
- Microsoft.CodeAnalysis.VisualBasic.Workspaces
- Microsoft.CodeAnalysis.EditorFeatures.Text
- Microsoft.VisualStudio.LanguageServices

That's at least the list from a quick scan of the internal repo; there's probably going a few more too but that's the "obvious" list.

Reply:
Let us get them added now